front-end/src/screens/Settings/email.tsx

```diff
@@ -95,43 +103,87 @@ const Email = ({ className }: Props): JSX.Element => {
 	};
 
 	return (
-		<Form standalone={false}>
-			<Form.Group className={className}>
+		<Form className={className} standalone={false} onSubmit={handleSubmit(handleSubmitForm)}>
+			<Form.Group>
 				<Form.Field width={10}>
 					<label>Email</label>
+					{editing ?
+						<input
+							className={errors.email ? 'error' : ''}
+							defaultValue={email || ''}
+							name='email'
+							placeholder='mail@example.com'
+							type='email'
+							ref={register({
+								pattern: /^[A-Z0-9_'%=+!`#~$*?^{}&|-]+([.][A-Z0-9_'%=+!`#~$*?^{}&|-]+)*@[A-Z0-9-]+(\.[A-Z0-9-]+)+$/i
```

Comment:
we should add a `required: true` here

Reply:
no user can delete his email also. In that case empty string will be there

Reply:
But this doesn't work then, with an empty email, the back is throwing "invalid email" from the `changeEmail` mutation: https://github.com/paritytech/polkassembly/blob/master/auth-server/src/resolvers/mutation/changeEmail.ts#L17.

Reply:
handling this case

Reply:
done